app/models/pieces/pawn.rb

```diff
@@ -1,2 +1,13 @@
 class Pawn < Piece
+  def valid_move?(x_destination, y_destination)
+    x_dist = (x_coord - x_destination).abs
+    y_dist = (y_coord - y_destination).abs
+
+    if y_coord == 1 || 6
```

Comment:
I don't think this line is doing what you expect.  I think this will always go through the if statement.  The reason being `y_coord == 1` will evaluate to a `true` or `false`.  Then it will evaluate:

```
if false || 6
```

6 is a "truthy" value so it will evaluate to true.